modules/install/join_rke2_master.sh

```diff
@@ -97,6 +97,53 @@ cis_setup() {
   fi
 }
 
+etcd_download() {
+    sudo curl -L https://github.com/etcd-io/etcd/releases/download/v3.5.0/etcd-v3.5.0-linux-amd64.tar.gz -o etcd-v3.5.0-linux-amd64.tar.gz
+    sudo tar xzvf etcd-v3.5.0-linux-amd64.tar.gz -C /usr/local/bin --strip-components=1 etcd-v3.5.0-linux-amd64/etcdctl
```

Comment:
Can we have the etcd version in a variable so that it's easier to update? (in all the places where we're doing this)